AdminUi/test/AdminUi.Tests.Integration/StepDefinitions/ClientsStepDefinitions.cs

```diff
@@ -7,17 +7,21 @@ namespace AdminUi.Tests.Integration.StepDefinitions;
 [Binding]
 [Scope(Feature = "GET Clients")]
 [Scope(Feature = "DELETE Clients")]
+[Scope(Feature = "PATCH Clients")]
 public class ClientsStepDefinitions : BaseStepDefinitions
 {
     private readonly ClientsApi _clientsApi;
     private string _clientId;
-    private HttpResponse<List<ClientDTO>>? _response;
+    private string _clientSecret;
+    private HttpResponse<List<ClientDTO>>? _clientsResponse;
+    private HttpResponse<ClientDTO>? _clientResponse;
```

Comment:
How about 

```suggestion
    private HttpResponse<List<ClientDTO>>? _getClientsResponse;
    private HttpResponse<ClientDTO>? _getClientResponse;
```

And additionally:
```c#
    private HttpResponse<List<CreateClientResponse>>? _createClientResponse;
    private HttpResponse<ChangeClientSecretResponse>? _changeClientSecretResponse;
```